metadata.json

```diff
@@ -58,7 +58,8 @@
       "operatingsystemrelease": [
         "7",
         "8",
-        "9"
+        "9",
+        "10"
```

Comment:
Perhaps this is a good excuse to start supporting CentOS Stream 10 in our CI for acceptance tests, though I doubt there are Puppet packages available though :(

Reply:
I can confirm there is no puppet agent for CS10.  I pulled the one from Fedora to test it.